bitwarden/mobile
# PS-518 - Add setting to block AppCenter / Analytics - Mobile

## Type of change

## Objective
Add a new entry in settings to allow user to disable sending information of crash reports to Appcenter.

## Code changes
* **SettingsPageViewModel.cs:** Added new method, `AppCenterReportingAsync`, to handle activation and deactivation of Crashes SDK. All changes are protected by compilation tags and shouldn't appear in **FDroid** build or **Debug** (Appcenter is only running in release compilations).
* **SettingsPage.xaml:** Add call to AppCenterReportingAsync 

## Testing Requirements
Note: This change will only not appear in **Debug** or **FDroid** builds.
1. Login into your account
2. Go to **Settings** tab
3. Scroll down to **Others** section
4. Click on **Report crash logs**

## Screenshots
<img width="408" alt="image" src="https://user-images.githubusercontent.com/4648522/168031785-51f3302c-e3ca-41c2-83fc-dffb3d7559cb.png">
<img width="396" alt="image" src="https://user-images.githubusercontent.com/4648522/168031820-873dfd48-825e-4aa8-9879-8e9a51122a97.png">

## Before you submit
